.github/PENDING_REVIEW_NOTIFICATION_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+Hi {{ username }},
```

Comment:
@SD-13 Could we use the updated message I have in this PR? https://github.com/oppia/oppia/pull/18553/files#diff-56bf299501545e9fd22a67dbb874edd4764f26fe0040dac349d06c95b5a9dc7eR1

Also just to check, is that file in the oppia/oppia repo still the source of truth, or is it this one? If one of them isn't used we should get rid of it. /cc @DubeySandeep

Reply:
Right now it is using the one in this repo, so we can get rid of the template in oppia/oppia.

Reply:
@seanlip I have used your message but with a small change. Also, as per the current work, this repo template will be used, so we can get rid of the one in `oppia/oppia` repo.